build.sbt

```diff
@@ -804,6 +804,7 @@ val allProjects = Seq(
   jsBridge06,
   jsBridge1,
   launcher,
+  launcherTest,
```

Comment:
Why do we need to add it here?

Reply:
because typing `compile` or `test` should also be run in `launcherTest` project. I forgot to add that last time and we have undetected errors in the project which were uncaught